core/resources/scene/config.yaml

```diff
@@ -16,6 +16,12 @@ textures:
     water-texture:
         url: normals.jpg
         filtering: nearest
+
+# provide global fontfile names to be used for this stylesheet
+# these could be bundled with the tangram repo or could be loaded from system path
+# priority is given to bundled fonts
+fonts: [ Roboto-Regular.ttf, FiraSans-Medium.ttf]
```

Comment:
These file names are platform-specific, we can't require them to be specified in the stylesheet (it defeats the whole purpose of having a cross-platform implementation of the renderer).